blazor/in-place-editor/events.md

```diff
@@ -118,7 +162,7 @@ This section explains the list of events of the In-place Editor's component whic
 
 ## Destroyed
 
-`Destroyed` event  triggers when the component gets destroyed.
+[Destroyed](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.InPlaceEditor.InPlaceEditorEvents-1.html#Syncfusion_Blazor_InPlaceEditor_InPlaceEditorEvents_1_Destroyed) event  triggers when the component gets destroyed.
```

Comment:
The [Destroyed](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.InPlaceEditor.InPlaceEditorEvents-1.html#Syncfusion_Blazor_InPlaceEditor_InPlaceEditorEvents_1_Destroyed) event triggers when the component gets destroyed.